### PR TITLE
Rc navigate maps

### PR DESCRIPTION
**Pull request by ryancoughlin**

### CODE REVIEW

File: app/components/map.js [hunk 1]
```diff
@@ -1,49 +1,39 @@
...
-
-      this.setState({
-        userLocation: { latitude, longitude },
-      })
-    })
-
-    fetch('http://localhost:8000/api/get-stations').then(res => res.json()).then((json) => {
-      this.setState({ stations: json.formattedStations })
-    })
+    if (!this.props.stations) {
```
Comment (ryancoughlin) [inline app/components/map.js]: I only need to do this once, no need to keep making the request.

File: app/components/station-detail/today-tide-row.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import BaseStyle from '../../base-styles'
 
 export default class NextTideRow extends Component {
   formatTideTime(time) {
-    return moment(time).format('hh:mma')
+    return moment.utc(time).local().format('hh:mma')
```
Comment (ryancoughlin) [inline app/components/station-detail/today-tide-row.js]: This was the fix @BlakeWilliams. I need to explicitly call `utc` there

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
 # Salty iOS
 
 Salty for iOS, built in React Native. Fetches tide and weather information.
+
+# To get up and running
+
+1. Install dependencies using `yarn`
+2. Run `pod install`
+3. Move the `AirGoogleMaps` folder to the Xcode project. You can get that folder from `node_modules/react-native-maps/lib/ios/`
```
Comment (ryancoughlin) [inline README.md]: This is some weird work around to get Google Maps working with React Native Maps. I didn't want to commit all those files, felt crazy to do that. Thoughts?

Comment (BlakeWilliams) [inline README.md]: I think you can just add it to header paths. I think we've done that before.

Comment (ryancoughlin) [inline README.md]: Ah ok, so you set some path to tell it that: this files lives here.

**Pull request merged**